packages/dart_frog/analysis_options.yaml

```
@@ -1 +1,5 @@
-include: package:very_good_analysis/analysis_options.6.0.0.yaml
+include: package:very_good_analysis/analysis_options.9.0.0.yaml
```

```suggestion
include: package:very_good_analysis/analysis_options.yaml
```
we started to leave the default file in `very_good_templates` since it already resolves to the latest version, we can do it here also since this is the latest

---

yup I don't have a strong preference. I'll update all of them to be consistent in a few mins 👍